apps/meteor/app/livechat/server/api/lib/inquiries.ts

```diff
@@ -39,24 +40,26 @@ export async function findInquiries({
 	pagination: { offset: number; count: number; sort: Record<string, number> };
 }): Promise<PaginatedResult<{ inquiries: Array<ILivechatInquiryRecord> }>> {
 	const department = await applyDepartmentRestrictions(userId, filterDepartment);
-
+	const defaultSort = await LivechatInquiry.getSortingQuery();
 	const options = {
 		limit: count,
-		sort: sort || { ts: -1 },
 		skip: offset,
+		sort: { ...sort, ...(defaultSort as object) },
 	};
 
-	const filter = {
-		...(status && { status }),
+	const filter: Filter<ILivechatInquiryRecord> = {
 		$or: [
 			{
 				$and: [{ defaultAgent: { $exists: true } }, { 'defaultAgent.agentId': userId }],
 			},
-			{ ...(department && { department }) },
+			{ ...((!!department && { department }) || {}) },
 			// Add _always_ the "public queue" to returned list of inquiries, even if agent already has departments
 			{ department: { $exists: false } },
 		],
 	};
+	if (status && status in LivechatInquiryStatus) {
+		filter.status = status as LivechatInquiryStatus;
```

Comment:
`status` is of type LivechatInquiryStatus. Why do you need this cast? 👀

Reply:
it was needed for some reason (TS complained), but it's not anymore :confused: